core/src/main/scripts/importer/validateData.py

```diff
@@ -981,6 +981,22 @@ class MutationsExtendedValidator(Validator):
 
     NULL_AA_CHANGE_VALUES = ('', 'NULL', 'NA')
 
+    # MAF values for Variant_Classification column
+    # from https://wiki.nci.nih.gov/display/TCGA/Mutation+Annotation+Format+%28MAF%29+Specification + Unknown:
+    VARIANT_CLASSIFICATION_VALUES = [
+       'Frame_Shift_Del',
+       'Frame_Shift_Ins',
+       'In_Frame_Del',
+       'In_Frame_Ins',
+       'Missense_Mutation',
+       'Nonsense_Mutation',
+       'Splice_Site',
```

Comment:
We might want to add some styling or functionality for `Splice_Region`. Maybe label it as **Splice** ?

<img width="1162" alt="screen shot 2017-05-26 at 11 19 23" src="https://cloud.githubusercontent.com/assets/9624990/26489108/55cdafa6-4206-11e7-9541-23dab348ed63.png">

`Splice_Site` looks find, and like this:

<img width="1167" alt="screen shot 2017-05-26 at 11 21 06" src="https://cloud.githubusercontent.com/assets/9624990/26489139/76809376-4206-11e7-8a95-7e6967c1795c.png">